jsonschema-validation.xml

```diff
@@ -701,30 +731,34 @@
                         <list style="hanging">
                             <t hangText="date-time:">
                                 A string instance is valid against this attribute if it is
-                                a valid representation according to the "date-time" production.
```

Comment:
@karenetheridge I'm confused, `duration` [already cites the ISO rule as quoted in the RFC](https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-validation-00#section-7.3.1).  If it's not in the RFC, we are not referencing it.  All we are doing is referencing that production.  That's why the word "production" was used.

I think "rule" (with the RFC section number, which is already there) is sufficient, although I would not object to ABNF so that's fine.

Reply:
I find the use of the word "production" to be very confusing and would prefer we avoid using it unless it's referencing something very specific, in which case we should link to its definition somewhere in the preamble like we do for MUST/MAY/RECOMMENDS etc.

Reply:
@karenetheridge did we not decide on "rule"?  Except for the XML one which already links to a document where it is used for that exact thing with the exact given name?

Also, I am sorry, but these are common terms in standards-ese.  One person's personal unfamiliarity with or dislike of a term is not a reason to avoid it.  It's a learning process to understand these documents fully, and the documents are not meant to be a tutorial in how to read them.  We can put a tutorial or glossary or whatever on the web site.

This comes up nearly every draft.  These are formal documents.  They are not for general audiences.  @jdesrosiers is doing a ton of work right now on the release notes and the Understanding JSON Schema book.  Those are for general audiences.  Those are where we should be mindful of accessible wording and be sure to define and document anything that might be confusing.

The specification is not for that.  The specification is for formal language, and understanding of the formal language (or at least the willingness to learn it) is assumed and required.